forecast/paper/supplement.Rmd

```diff
@@ -801,6 +875,85 @@ plotter_hotspots(hotspots_honest)
 \clearpage
 
 
+<!-- Cumulative Mean -->
+
+```{r cumulative-mean, fig.cap="This figure shows the cumulative sum of WIS for each forecaster divided by the cumulative sum of WIS for the baseline model. The shaded background shows" }
```

Comment:
I believe "mean" is meant here rather than "sum".  Also, shaded background description is not finished.

Reply:
(finished the background description)